src/array/data_chunk.rs

```diff
@@ -115,6 +115,18 @@ impl DataChunk {
     pub fn header(&self) -> Option<&[String]> {
         self.header.as_deref()
     }
+
+    pub fn from_rows<'a>(rows: &[RowRef<'a>], chunk: &Self) -> Self {
```

Comment:
I prefer to pass a schema here, instead of a chunk. Add a TODO here?

Reply:
Yes I have the same idea. It will be more elegant 🥰
```suggestion
    /// TODO: pass a schema here, instead of a chunk
    pub fn from_rows<'a>(rows: &[RowRef<'a>], chunk: &Self) -> Self {
```